lib/sshutils/server.go

```diff
@@ -417,6 +418,14 @@ func (s *Server) HandleConnection(conn net.Conn) {
 	wrappedConn := wrapConnection(wconn, s.log)
 	sconn, chans, reqs, err := ssh.NewServerConn(wrappedConn, &s.cfg)
 	if err != nil {
+		// Ignore EOF as these are triggered by loadbalancer health checks
+		if !errors.Is(err, io.EOF) {
+			s.log.
```

Comment:
I've introduced a log message here. Sinking this error previously was what meant that it was difficult to work out what was up with the incoming connections.